src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -723,6 +724,9 @@ internal abstract partial class BaseBinaryOperatorExpression : Operation, IHasOp
         /// Operation method used by the operation, null if the operation does not use an operator method.
         /// </summary>
         public IMethodSymbol OperatorMethod { get; }
+
+        public bool IsLifted { get; }
```

Comment:
The rest of the implementations of the properties have their doc comments copied over, we should stay consistent here.